prow/overlays/cnv-prod/labels_operate-first.yaml

```diff
@@ -312,6 +361,18 @@ default:
         - name: wontfix
       target: both
       addedBy: humans
+    - color: fc05d7
+      description: Indicates an issue/PR is related to end-user documentation.
+      name: users/documentation
```

Comment:
This might be equivalent to `kind/documentation` ?

Reply:
I think it depends on the full scope of `kind/documentation`, but I suspect there is some overlap and some not.

From a project perspective, we will always have a set of users and a growing body of documentation for them. The definition of 'users' includes:  open source projects doing QE; people on learning pathways (SRE, data scientist, etc.); people who have other workloads on the system for any reason (running demos, etc.) Thus the purpose of the `community/documentation` label is to be solely for documentation-related issues/PRs that are intended to be part of the user-facing documentation. (I am intentionally not limiting it to written content, as it may be video, audio, and other artifacts, and it may be related to tooling and automation to bring the learning pathways to life—anything being done in the name of "end-user documentation".) 

I suspect that `kind/documentation` covers documentation for the running cloud environment and other cases. If that's the case, then `community/documentation` is a subset of that set, but with a twist:

It's almost certain that some portion of the "documentation for keeping the cloud environment running" will also be auto-included in the learning pathways for "people learning how this thing is running" and especially for "people learning hands-on how this is running by submitting a PR on the live system they can watch get tested, accepted, and deployed." But I don't think all of the docs and tooling that might be `kind/documentation` is going to be part of end-user documentation, so there is value in having two distinct but intertwinable labels.

Reply:
Right, got it. What about combining labels? `kind/documentation` `area/community` is for what you intended, `kind/bug` `kind/documentation` could be for doc bugs, and `kind/feature` `area/community` could be used for a new feature that we want to implement for the community?

In the end, its all up to you... To me it just felt handy to reuse what we got, and as subjects get contextualized in different ways, multiple labels might be a good solution.

wdyt?

Reply:
OK, I think I see where you are going. I definitely want to create metadata that fits the format of other labels and helps machine learning, right?

I'll rework the PR along these lines and see how it looks, thanks.

Reply:
@goern OK, these labels seem _way_ more useful. Did I get the idea correct?

To make it clear for any watchers, what this does is make it so any issue can take an `area` label to mark that it's important to the uses, contributors, or community overall. Similarly, the `kind` labels can mark issues of that kind all over, which can be related or not related to an `area`. This makes those labels more useful to be applied to other areas, e.g. `kind/news` & `area/releases` & `kind/bot` might label an issue that can be brought in as news about updates to a bot.